src/api.rs

```diff
@@ -363,6 +375,7 @@ impl Default for MatrixCoefficients {
     }
 }
 
+/// Contains different types of the color primaries, they defines how the chromaticy coordinates of the source picture
```

Comment:
```suggestion
/// Contains different types of the color primaries, they defines how the chromaticity coordinates of the source picture
```

The grammar is a bit awkward overall.